extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -107,8 +107,14 @@ export class AmpAdApiHandler {
       // If support render-start, create a race between render-start no-content
       this.adResponsePromise_ = listenForOncePromise(this.iframe_,
         ['render-start', 'no-content'], this.is3p_).then(info => {
-          if (info.data.type == 'render-start') {
-              //report performance
+          const data = info.data;
+          if (data.type == 'render-start') {
+            const opt_data = data.opt_data;
+            if (opt_data) {
```

Comment:
I decided to not wrap `opt_data` during postMessage, which makes attributes in `opt_data` to be added to `data` object. (e.g `data.width`/`data.height`). 
I don't think we need the check here in that case, since `updateSize_()` will check if the width/height is `undefined`